src/services/kratos/auth-phone-no-password.ts

```diff
@@ -189,8 +281,10 @@ export const AuthWithPhonePasswordlessService = (): IAuthWithPhonePasswordlessSe
 
   return {
     login,
```

Comment:
now we're introducing cookie, this function should be also probably updated as `loginToken`